techniques/general/G9.html

```diff
@@ -51,102 +49,89 @@
   <ul>
   
     <li>
       
 <a href="https://w3c.github.io/wcag/understanding/captions-live">1.2.4: Captions (Live)</a>
-(<strong>Sufficient</strong>, together with <a href="../general/G93">G93: Providing open (always visible) captions</a>)
+(<a href="../general/G93">G93: Providing open (always visible) captions</a> とともに<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
 <a href="https://w3c.github.io/wcag/understanding/captions-live">1.2.4: Captions (Live)</a>
-(<strong>Sufficient</strong>, together with <a href="../general/G87">G87: Providing closed captions</a>)
+(<a href="../general/G87">G87: Providing closed captions</a> とともに<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
 <a href="https://w3c.github.io/wcag/understanding/captions-live">1.2.4: Captions (Live)</a>
-(<strong>Sufficient</strong>, together with <a href="../general/G87">G87: Providing closed captions</a>
-  using a more specific technique)
+(より具体的なテクニックを用いれば <a href="../general/G87">G87: Providing closed captions</a> とともに<strong>十分なテクニック</strong>)
     </li>
   
   </ul>
   
 
 
 
-  <p>This technique applies to all technologies that present audio visual information.</p>
+  <p>このテクニックは、視聴覚情報を提示する全ての技術に適用される。</p>
 
 </div>
 </section>
 	
 
-<section id="description"><h2>Description</h2>
-      <p>The objective of this technique is to allow users who cannot hear to be able
-            to access real-time synchronized media broadcasts in the same <a href="https://w3c.github.io/wcag/guidelines/22/#dfn-human-language-s" target="terms">human language</a> as the orginal. It is more difficult to create
-            accurate real-time captions because there is little time to correct mistakes
-            or to listen a second time or consult someone to be sure the words are
-            accurately reproduced. It is also harder to simplify or paraphrase
-            information if it is flowing too quickly.</p>
-      <p>Real-time typing text entry techniques exist using stenographic and rapid
-            typing technologies. Re-voicing speech-to-text (where a person listens to
-            speech and then carefully re-voices it into a computer trained to their
-            speech) is used today for telephone relay services and may be used in the
-            future for captioning. Eventually speech-to-text with correction will be
-            possible.</p>
-   </section><section id="examples"><h2>Examples</h2>
-       <section class="example" id="example-1"><h3>Example 1</h3>
-		   <p>A television studio uses a real-time captioning service
-                to create captions for its evening news online.</p>
+<section id="description"><h2>解説</h2>
+      <p>このテクニックの目的は、音声を聞くことができない利用者が、オリジナルと同じ<a href="https://waic.jp/translations/WCAG22/#dfn-human-language-s" target="terms">自然言語</a>リアルタイムに同期したメディア放送にアクセスできるようにすることである。間違いを訂正する又は、二度聞きもしくは単語が正確に再現されていることを確認するために誰かの意見を聞く時間はほとんどないので、正確なリアルタイムのキャプションを作成することはより困難である。情報があまりに速く流れている時は、簡略化又は言い換えもさらに困難である。</p>
```

Comment:
in the same human language as the orginal の訳を少し調整しました。

```suggestion
      <p>このテクニックの目的は、音声を聞くことができない利用者が、オリジナルと同じ<a href="https://waic.jp/translations/WCAG22/#dfn-human-language-s" target="terms">自然言語</a>で、リアルタイムに同期したメディア放送にアクセスできるようにすることである。間違いを訂正する又は、二度聞きもしくは単語が正確に再現されていることを確認するために誰かの意見を聞く時間はほとんどないので、正確なリアルタイムのキャプションを作成することはより困難である。情報があまりに速く流れている時は、簡略化又は言い換えもさらに困難である。</p>
```